### PR TITLE
Add a `bundle install` test

### PR DESCRIPTION
**Pull request by jkeiser**
I am mainly doing this to find out if Travis is broked

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,7 @@
+bin/
+!bin/git-generate-changelog
+!bin/github_changelog_generator
```
Comment (olleolleolle) [inline .gitignore]: Should these really go in here?

**Comment by olleolleolle** [comment]
I like this test - and I had a question about one of the gitignores.

**Comment by jkeiser** [comment]
Anyone who uses bundle install --binstubs will put lots of binaries in there. Only reason for the gitignore

**Comment by olleolleolle** [comment]
Thanks, grand! :+1:

**Pull request merged**